packages/main-ui/src/pages/contents/_id/minting_popup.rs

```diff
@@ -111,15 +110,15 @@ pub fn MintingPopup(lang: Language, id: ReadOnlySignal<i64>) -> Element {
             }
 
             div { class: "carousel-item relative w-full", id: "slide3",
-                div { class: "w-full flex flex-col justify-between items-center gap-[20px] pt-[40px]",
+                div { class: "w-full flex flex-col justify-between items-center gap-[20px] pt-[40px] p-4",
                     CompleteIcon { size: 120 }
                     div { class: "flex flex-col w-full items-center justify-center gap-[20px] text-[#5B5B5B]",
 
                         p { class: "w-full text-center text-[15px] tracking-wide text-[#5B5B5B] font-bold",
                             "{tr.complete_text}"
                         }
                         button {
-                            class: "flex justify-center items-center rounded-[12px] w-[440px] h-[50px] font-normal text-[18px] bg-[#24B28C] hover:bg-[#1E9E7A] text-white",
+                            class: "flex justify-center items-center rounded-[12px] w-full max-w-[440px] h-[50px] font-normal text-[18px] bg-[#24B28C] hover:bg-[#1E9E7A] text-white", // Use max-width for responsiveness
```

Comment:
```suggestion
                            class: "flex justify-center items-center rounded-[12px] w-full h-[50px] font-normal text-[18px] bg-[#24B28C] hover:bg-[#1E9E7A] text-white", // Use max-width for responsiveness
```

even if it contains max-w, it would be inherited width of parent maybe.